package.json

```diff
@@ -15,7 +15,8 @@
 	},
 	"scripts": {
 		"test": "xo && nyc ava",
-		"lint": "node ./test/lint/lint.js",
+		"test-coverage": "nyc --reporter=lcov ava",
+		"lint": "node ./test/lint/lint.js && xo",
```

Comment:
XO is now run twice, both on `test` and `lint`.

Reply:
@sindresorhus Yes and no. In the GitHub CI pipe `test-coverage` is run rather than `test`. Partly to get coverage report and partly to move the `xo` part from being run on every Node.js version to being run just once as part of the linting flow.

So, `test.yml` runs `test-coverage` while `other.yml` (suggestion on better name?) runs `lint` and `integration`

I thought it was best to leave the `test` as is, as that's likely the one that gets run manually the most.